documentation/docs/identus/cloud-agent/authentication.md

```diff
@@ -125,7 +125,7 @@ When the JWT Token authentication is enabled, it includes protection for all mul
 Two mechanisms get utilized in JWT authorization:
 
 1. __Role-based authorization__  
-   It implements this [ADR](https://staging-docs.atalaprism.io/adrs/adr/20240103-use-jwt-claims-for-agent-admin-auth/), which can authorize both the administrator and tenant role.
+   It implements this [ADR](https://staging-docs.atalaprism.io/hyperledger/identus/adrs/adr/20240103-use-jwt-claims-for-agent-admin-auth/), which can authorize both the administrator and tenant role.
```

Comment:
```suggestion
   It implements this [ADR](https://staging-docs.atalaprism.io/adrs/adr/20240103-use-jwt-claims-for-agent-admin-auth/), which can authorize both the administrator and tenant role.
```

Reply:
This is not changed yet. @CryptoKnightIOG , may you use the suggestion?